pkg/deployment/agent_test.go

```diff
@@ -158,6 +158,44 @@ func TestAgentLabels(t *testing.T) {
 	assert.Equal(t, fmt.Sprintf("%s-agent", a.jaeger.Name), dep.Spec.Template.Labels["app.kubernetes.io/name"])
 }
 
+func TestAgentDNSPolicyAndHostNetwork(t *testing.T) {
+	jaeger := v1.NewJaeger(types.NamespacedName{Name: "TestAgentDNSPolicyAndHostNetwork"})
```

Comment:
for new tests, we are using `my-instance` instead.

Reply:
I will fix it today.